polygon/websocket/__init__.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         subscriptions: Optional[List[str]] = None,
         max_reconnects: Optional[int] = 5,
         secure: bool = True,
+        custom_json: Optional[Any] = None,
```

Comment:
If there's no builtin type for `json` can you make one that has `loads` and `dumps` methods?

Reply:
I prefer 1.